app/presenters/two_factor_authentication/sms_selection_presenter.rb

```diff
@@ -4,6 +4,15 @@ def method
       :sms
     end
 
+    def info
```

Comment:
Should we get rid of the default `info` method implementation in `PhoneSelectionPresenter`, since we're expecting that each subclass should define a specific text?

Reply:
After reviewing further, I think I understand now that the intention is that the "Phone" presenter would be offered during sign-up, whereas the individual presenters would be offered during sign-in? 

In which case, maybe we should then at least remove the `configurations.any?` condition path of the base `PhoneSelectorPresenter`?

Reply:
Yep Phone presenter would be during sign up and the others during sign-in.